app/src/main/java/adhoclabs/co/burnersamplesdk/SampleActivity.java

```diff
@@ -29,6 +31,7 @@ protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_sample);
     burnerSDK = BurnerSDK.getInstance();
+    Fabric.with(this, new Crashlytics());
```

Comment:
This should go in the `Application` subclass, so you're not constrained to this activities context.